setup.py

```diff
@@ -53,7 +53,7 @@
     author_email="gpkit@mit.edu",
     url="https://www.github.com/hoburg/gpkit",
     install_requires=["numpy >= 1.8.1", "pint", "scipy"],
-    version="0.5.0.0",
+    version="0.5.0.2",
```

Comment:
I agree with @galbramc that we should use the PyPI testing site to avoid all these minor versions getting pushed to the official site.